models/encoding.go

```diff
@@ -13,6 +18,7 @@ type Encoding struct {
 	CloudRegion      bitmovintypes.CloudRegion    `json:"cloudRegion,omitempty"`
 	Status           string                       `json:"status,omitempty"`
 	InfrastructureID *string                      `json:"infrastructureId,omitempty"`
+	Infrastructure   *Infrastructure              `json:"infrastructure,omitempty"`
```

Comment:
This one has to be a pointer otherwise when we don't specify an Infrastructure we end up with `infrastructure: {}` in the JSON (even with `omitempty` enabled).